platforms/cortex/platform.cpp

```diff
@@ -36,9 +36,6 @@
 #define ENABLE_DEBUG        (0)
```

Comment:
in commit `sys: shell_menu: factorize handling of Protobuf commands`:
Move code handle => Move code **to** handle